lib/portlayer/storage/volume/cache/cache.go

```diff
@@ -35,17 +37,17 @@ type VolumeLookupCache struct {
 	// Maps IDs to Volumes.
 	//
 	// id -> Volume
-	vlc     map[string]Volume
+	vlc     map[string]volume.Volume
 	vlcLock sync.RWMutex
 
 	// Maps the service url of the volume store to the underlying data storage implementation
-	volumeStores map[string]VolumeStorer
+	volumeStores map[string]volume.VolumeStorer
 }
 
 func NewVolumeLookupCache(op trace.Operation) *VolumeLookupCache {
 	v := &VolumeLookupCache{
-		vlc:          make(map[string]Volume),
-		volumeStores: make(map[string]VolumeStorer),
+		vlc:          make(map[string]volume.Volume),
```

Comment:
Is there a reason why we use Volume as a value, not as a pointer? Volume is a relatively massive data structure, copying it each time is not really desirable.